src/T2DMap.cpp

```diff
@@ -5078,7 +5078,7 @@ void T2DMap::resizeMultiSelectionWidget()
     if (mMultiSelectionListWidget.topLevelItemCount() > 0) {
         QTreeWidgetItem* rowItem = mMultiSelectionListWidget.topLevelItem(1);
         // The following factors are tweaks to ensure that the widget shows all
-        // the rows, as the header seems bigger than the value returned, statics
+        // the rows, as the header seems bigger than the value returned, statistics
```

Comment:
Nope, the term is `static` and it is pluralised.

Reply:
Static values, might read better

Reply:
To be change, reads wrong